src/mds_array_manipulation/__init__.py

```diff
@@ -1,4 +1,2 @@
-# read version from installed package
-from importlib.metadata import version
```

Comment:
We don't need this import anymore so removed it

Reply:
In my opinion, we should keep it as when we change the version so this will change according to our latest version.
Does version change it according to our release?

Reply:
We discussed during the meeting but we were unable to get this working with pytest -- it complains about import errors if we try to automatically resolve the version. I see in Tiff's package as well she's hardcoded it. If you're able to get it working through let us know

Reply:
Sorry I forgot what we discussed. I 'll resolve the conversation now. Sorry again.